README.md

```diff
@@ -914,6 +914,78 @@ class ApplicationJob < ActiveJob::Base
 end
 ```
 
+### Querying Jobs
+
+Jobs are contained in the mounted GoodJob::Job table. The schema is:
+
+```
+GoodJob::Job < GoodJob::Execution {
+                     :id => :uuid,
+             :queue_name => :text,
+               :priority => :integer,
+      :serialized_params => :jsonb,
+           :scheduled_at => :datetime,
+           :performed_at => :datetime,
+            :finished_at => :datetime,
+                  :error => :text,
+             :created_at => :datetime,
+             :updated_at => :datetime,
+          :active_job_id => :uuid,
+        :concurrency_key => :text,
+               :cron_key => :text,
+    :retried_good_job_id => :uuid,
+                :cron_at => :datetime
+}
```

Comment:
We're a few versions behind on GoodJob. If I didn't get the schema quite right with the current branch, please either let me know or put in the proper, up-to-date schema.